taca/utils/misc.py

```diff
@@ -109,6 +110,25 @@ def days_old(date, date_format="%y%m%d"):
         return None
     return time_dif.days
 
+
+def to_seconds(days=None, hours=None):
+    """ Convert given day/hours to seconds and return
+
+        :param int days: days to be converted
+        :param int hours: hours to be converted
+    """
+    #only either days or hours should be speciefied, but atleast one should be specified
+    if days and hours:
+        raise SystemExit('Both "days" and "hours" were given, use only either of them')
+    elif not days and not hours:
```

Comment:
@vezzi : Made a new [commit](https://github.com/SciLifeLab/TACA/pull/173/commits/b79d9f04b3481ff7bd31adc62fe4dfb61b7fbf49), it takes care of the issue we discussed :+1: 

**Sample output:**
```
$ taca storage --days 0 cleanup --site nas
Usage: taca storage [OPTIONS] COMMAND [ARGS]...

Error: Invalid value for "-d" / "--days": 0 is smaller than the minimum valid value 1.

$ taca storage --hours 0 cleanup --site nas
Usage: taca storage [OPTIONS] COMMAND [ARGS]...

Error: Invalid value for "-h" / "--hours": 0 is smaller than the minimum valid value 1.

$ taca storage --days 5 --hours 0 cleanup --site nas
Usage: taca storage [OPTIONS] COMMAND [ARGS]...

Error: Invalid value for "-h" / "--hours": 0 is smaller than the minimum valid value 1.

$ taca storage --days 5 --hours 10 cleanup --site nas
Both "days" and "hours" were given, use only either of them

$ taca storage cleanup --site nas
provide either "days" or "hours"
```